### PR TITLE
Fix: Auto linking for RN >= 0.69

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
Remove the removed config from the config file.
This is a breaking change.

## :bulb: Motivation and Context
Fixes https://github.com/getsentry/sentry-react-native/issues/2267

## :green_heart: How did you test it?
Running it

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [ ] No breaking changes

## :crystal_ball: Next steps
Patch https://github.com/getsentry/sentry-wizard

### CODE REVIEW

**Comment by marandaneto** [comment]
@bruno-garcia What if we say that the RN SDK 5.x requires RN version >= 0.69?
We could technically drop stuff like https://github.com/getsentry/sentry-react-native/pull/2015/
It'd be easier to maintain as well since we don't need to test stuff on older versions, we'd need to patch bug fixes on v4.x when needed though.
Ideally not the best solution for the user, but this is a breaking change and auto-linking won't work for either before or after 0.69.

**Comment by bruno-garcia** [comment]
> 

Sounds like it's it's time. Can we spit out build warnings/errors in case the expected version isn't match? My biggest worry is of users getting unrelated error messages, being frustrated and/or raising issues with support. 

If we can break at build time, with:

Sentry 5.0 on older RN: `The Sentry SDK version 5.0.0 and above requires React Native 0.69 or higher`
Sentry 4.x on older RN: `The Sentry SDK version 4 doesn't support React Native 0.69. Please upgrade to Sentry 5.0.0 or higher`

**Comment by marandaneto** [comment]
TODO: `yarn install` after sentry-wizard gets released to update the yarn.lock file.

File: yarn.lock [hunk 1]
```diff
@@ -1255,12 +1257,12 @@
...
-  version "1.2.17"
-  resolved "https://registry.yarnpkg.com/@sentry/wizard/-/wizard-1.2.17.tgz#c3247b47129d002cfa45d7a2048d13dc6513457b"
-  integrity sha512-wXzjOZVDzh+1MhA9tpZXCKNTDusL2Dc298KGQkEkNefbW+OlYbsWwF7GAihLKUjYOSnKKZFzLIwbD+gxwxWxlw==
+"@sentry/wizard@2.0.0":
+  version "2.0.0"
+  resolved "https://registry.yarnpkg.com/@sentry/wizard/-/wizard-2.0.0.tgz#94bd4d0eeaae67b9577bd1fe2688b2be9d1f77e9"
+  integrity sha512-v6rOlJE0bOGSuekd+LTitofObIXKxtrQW3pXG2mwFVPQ6wezpFpUd8D/5hLJntLSvon/vFkQu43WvU+0VNBR3Q==
   dependencies:
-    "@sentry/cli" "^1.52.4"
+    "@sentry/cli" "^1.72.0"
```
Comment (bitsandfoxes) [inline yarn.lock:1265]: is there a specific reason you stick to 1.72.0?

File: yarn.lock [hunk 2]
```diff
@@ -1173,16 +1173,18 @@
     proxy-from-env "^1.1.0"
     which "^2.0.2"
 
-"@sentry/cli@^1.52.4":
-  version "1.57.0"
-  resolved "https://registry.yarnpkg.com/@sentry/cli/-/cli-1.57.0.tgz#87237ec6b77cb7582f018b580e9bb9481a809baa"
-  integrity sha512-lgfSQMRVcf00IkHkmqY6L9pv1uOEKDKbEjFQu2qstjg3kycswo1prz60xORJVpfr4qXDzqbSsAKEBp4VKG+ilw==
+"@sentry/cli@^1.72.0":
```
Comment (bitsandfoxes) [inline yarn.lock:1176]: Are you sticking with `1.72.0` because of the wizard dependency?

**Review by bitsandfoxes — APPROVED** [review]

File: yarn.lock (hunk 2 above)
Comment (marandaneto) [reply yarn.lock:1176]: Yeah, that's the min. version of the CLI for the newer wizard, but the `^` means anything above it, it's not 1.72 only.

**Comment by SiSa68** [comment]
With this version, the source map is not uploaded automatically for my application (with react-native 69.4 and in debug mode)
Is this a known issue?

**Comment by Acetyld** [comment]
Any idea on release date?

**Pull request merged**